src/main/src/xyz/dommi/beans/BetGameBean.java

```diff
@@ -0,0 +1,78 @@
+package xyz.dommi.beans;
+
+import com.mongodb.BasicDBObject;
+import com.mongodb.DB;
+import com.mongodb.DBObject;
+import xyz.dommi.db.BetGameDB;
+import xyz.dommi.db.DBConnection;
+import xyz.dommi.db.DBManager;
+import xyz.dommi.requests.Response;
+import xyz.dommi.requests.ResponseType;
+
+public class BetGameBean {
+    private String id;
+    private String userId;
+    private int answer;
+
+
+    public BetGameBean(String id, String userId) {
+        this.id = id;
+        this.userId = userId;
+        if((id != null && !id.equals(""))){
+            DBConnection dbConnection = new DBConnection();

```

Comment:
Kleiner Verbesserungsvorschlag für nachher: Vielleicht `DBConnection` ein mal in der Klasse instanzieren und die Instanz nutzen? Würde etwas Redundanz ersparen

Reply:
wäre sinnvoll ^^